bftengine/src/bftengine/ViewsManager.cpp

```diff
@@ -341,7 +341,7 @@ void ViewsManager::computeCorrectRelevantViewNumbers(ViewNum* outMaxKnownCorrect
   *outMaxKnownAgreedView = myLatestPendingView;
 
   size_t numOfVC = 0;
-  ViewNum* viewNumbers = reinterpret_cast<ViewNum*>(alloca(N * sizeof(ViewNum)));
+  std::vector<ViewNum> viewNumbers(N * sizeof(ViewNum));
```

Comment:
Here the parameter should be just N instead of sizeof(ViewNum). Alloca accepts size in bytes, while the vector - element count.

Reply:
Fixed.